x-sum.py

```diff
@@ -0,0 +1,38 @@
+t= int(input())
+ 
+def x_sum(grid, m, n):
+      
+      dig_sum1= [0 for i in range(n+m-1)]
+      dig_sum2= [0 for i in range(n+m-1)]
+      
+      for i in range(m):
+            for j in range(n):
+                  sum_= i + j
+                  diff= j-i
+                  
+                  dig_sum1[sum_] += grid[i][j]
+                  if diff < 0:
+                        diff= n-1+abs(diff)
+                  dig_sum2[diff] += grid[i][j]
+      res= 0
+      for i in range(m):
+            for j in range(n):
```

Comment:
try to use expressive names instead of i and j   like use row and column

Reply:
i and j are universal matrix indicators I prefer using i and j since they are known and to not pollute the code with long variable names for indexing